src/client/src/services/achievemnts-api-service.js

```diff
@@ -0,0 +1,9 @@
+const URL = "http://localhost:8080";
+export default class AchievemntsApiService {
+  static async getAchievements(id) {
+    const response = await fetch(`${URL}/matching/trivia-answers/${id}`);
```

Comment:
we need to use await axios.get(`/matching...`) instead of fetch, because in production we can't pass localhost link